action.yml

```diff
@@ -0,0 +1,69 @@
+---
+name: mcaf-mcvs-golang-action
+description: |
+  The Mission Critical Acceleration Framework (MCAF)
+  Mission Critical Vulnerability Scanner (MCVS) Golang action.
+inputs:
+  golang-version:
+    description: |
+      The Golang version that has to be installed and used.
+    default: 1.21.5
```

Comment:
What I recently did is using the following in setup go:
```YAML
with:
          go-version-file: 'go.mod'
          ```
          This omits the need for manually maintaining the Go version